typescript/cli/src/utils/chains.ts

```diff
@@ -31,9 +32,10 @@ export async function runSingleChainSelectionStep(
 
 export async function runMultiChainSelectionStep(
   customChains: ChainMap<ChainMetadata>,
+  chainsToFilterOut: ChainName[] = [],
```

Comment:
Nit, I'd make this the 3rd param since we seem to use it less often than the 2nd optional param